backend/app.py

```diff
@@ -310,17 +338,42 @@ def get_user():
             "message": "User not found"
         }
     else:
-        return jsonify({
-            "success": True,
-            "message": "User found",
-            "data": {
-                "id": user.id,
-                "name": user.name,
-                "email": user.email,
-                "role": user.role,
-                "points_total": user.points_total
-            }
-        })
+        # check if this is a fellow inquiring about their point total,
+        # or if this is an admin inquiring about a fellow's total.
+        if user.role == "admin":
+            # get the specified info for admin
+            u_discord_id = request.args.get('name')
```

Comment:
is this the discord_id or the discord username? Just wanted to clarify because naming might be confusing here

Reply:
You’re right. I’ll rename the variables so it reflects usage from other APIs